### DISCUSSION TITLE
「你所提交的反馈无效」的触发条件？

### DISCUSSION THREAD

**Discussion by Pigeon0v0** [Meloong-Git/PCL#1609] [提问] [answered]
![image](https://user-images.githubusercontent.com/60414767/230717818-64f88281-9d40-4345-b139-056e35a101d7.jpeg)

上图情况出现有什么条件呢？

**Comment by LTCatt** [comment, marked as answer]
Bot 自动删除的条件很复杂，和提交的密度和频率、Tag、内容丰富程度、账号活跃度、是否有敏感词等等都有关。
主要是为了防止短时间恶意提交大量恶意反馈……
